euclidlib/photo/_le3_2pcf_wl.py

```diff
@@ -1,37 +1,154 @@
 from __future__ import annotations
 
 from os import PathLike
-import fitsio
-from euclidlib.photo import photo_data
+from typing import TYPE_CHECKING, Tuple, Union
 
+import fitsio  # type: ignore [import-not-found]
+from numpy.typing import NDArray
 
-def correlation_functions(path: str | PathLike[str]) -> dict:
+
+# type alias
+_DictKey: "TypeAlias" = Union[str, int, Tuple["_DictKey", ...]]
+
+def _key_from_string(s: str) -> _DictKey:
+    """return key from string representation"""
```

Comment:
Hi @gcanasherrera ,
maybe the description of the functions should start with a capital letter (e.g. `Return` rather than `return`?